pkgs/top-level/python-packages.nix

```diff
@@ -23203,6 +23203,28 @@ in modules // {
     };
   };
 
+  you-get = buildPythonPackage rec {
```

Comment:
`buildPythonApplication` is a convenient alternative; it's the same thing except it sets `namePrefix = ""` for you ...